### ISSUE TITLE
Exakat crash, no output even on `doctor`.

### ISSUE THREAD

**Issue by dotancohen** [exakat/exakat#77]
I've installed Exakat on Ubuntu Linux with the instructions from here:
https://exakat.readthedocs.io/en/latest/Installation.html#full-installation-with-debian-ubuntu

And using the Gremlin 3 installation instructions from here:
https://www.exakat.io/up-and-running-with-exakat/

When I run `doctor` there is no output:
```
$ php exakat.phar doctor

$ echo $?
255

$ uname -a
Linux latitude 4.15.0-36-generic #39-Ubuntu SMP Mon Sep 24 16:19:09 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ php -v
PHP 7.2.11
(abbreviated)
```

I know that this is not a support forum, but as I've had Exakat working in the past (on CentOS) and I've followed the installation instructions, I suppose that I should file an issue here [similar to this previous no-errors crash](https://github.com/exakat/exakat/issues/50).

**Comment by dotancohen** [comment]
It turns out that `config/exakat.ini` had bogus entries for the PHP executable locations' entires. For instance, even though I do not have PHP 7.3 installed, the line `php73 = /usr/local/sbin/php73` was found in that file.

Additionally, as I have PHP 7.2 installed, the PHP 7.2 version had two lines. One was the bogus line as described above, the second was the real installed path.

The `config/exakat.ini` file was created by running the `doctor` command. Here is the full output demonstration the creation of that file:
```
$ cat config/exakat.ini
cat: config/exakat.ini: No such file or directory

$ php exakat.phar doctor
exakat : 
    executable           : exakat.phar
    version              : 1.4.9
    build                : 822
    exakat.ini           : config/exakat.ini
    graphdb              : gsneo4j
    reports              : Ambassador
    themes               : CompatibilityPHP53,
                           CompatibilityPHP54,
                           CompatibilityPHP55,
                           CompatibilityPHP56,
                           CompatibilityPHP70,
                           CompatibilityPHP71,
                           CompatibilityPHP72,
                           CompatibilityPHP73,
                           CompatibilityPHP74,
                           Dead code,
                           Security,
                           Analyze,
                           Preferences,
                           Appinfo,
                           Appcontent
    extra themes         : 
    tokenslimit          : 1 000 000

PHP : 
    binary               : 7.2.11-2+ubuntu18.04.1+deb.sury.org+1
    memory_limit         : -1
    ext/curl             : Yes
    ext/hash             : Yes
    ext/phar             : Yes
    ext/sqlite3          : Yes
    ext/tokenizer        : Yes
    ext/mbstring         : Yes
    ext/json             : Yes
    ext/xmlwriter        : Yes

java : 
    installed            : Yes
    type                 : OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.2)
    version              : openjdk
    $JAVA_HOME           : 
    $JAVA_OPTIONS        : 

tinkergraph : 
    installed            : No (folder : tinkergraph)

gsneo4j : 
    installed            : No (folder : tinkergraph)

folders : 
    projects folder      : Yes
    projects/test        : No
    projects/default     : No
    projects/onepage     : No

php52 : 
    configured           : No

php53 : 
    configured           : No

php54 : 
    configured           : No

php55 : 
    configured           : No

php56 : 
    configured           : No

php70 : 
    configured           : No

php71 : 
    configured           : No

php72 : 
    configured           : Yes (/usr/bin/php7.2)
    version              : 7.2.11-2+ubuntu18.04.1+deb.sury.org+1
    short_open_tags      : Yes
    tokenizer            : Yes
    assertions           : Yes
    memory_limit         : -1
    timezone             : 
    token_version        : Php72

php73 : 
    configured           : No

php74 : 
    configured           : No

$ cat config/exakat.ini
 use tinkergraph or gsneo4j
graphdb = ''

 where is tinkergraph host
tinkergraph_host     = '127.0.0.1'
tinkergraph_port     = '8182'
tinkergraph_folder   = 'tinkergraph'

 where is neo4j inside a gremlin server host
gsneo4j_host     = '127.0.0.1'
gsneo4j_port     = '8182'
gsneo4j_folder   = 'tinkergraph'

 where is janusgraph host (alpha stage, use with caution)
janusgraph_host     = '127.0.0.1'
janusgraph_port     = '8182'
janusgraph_folder   = 'janusgraph'

php52 = 
php53 = 
php54 = 
php55 = /usr/local/sbin/php55
php56 = /usr/local/sbin/php56
php70 = /usr/local/sbin/php70
php71 = /usr/local/sbin/php71
php72 = /usr/local/sbin/php72
php73 = /usr/local/sbin/php73
php72 = /usr/bin/php7.2

token_limit = 1000000000

 Default themes to run
project_themes[] = 'CompatibilityPHP53'
project_themes[] = 'CompatibilityPHP54'
project_themes[] = 'CompatibilityPHP55'
project_themes[] = 'CompatibilityPHP56'
project_themes[] = 'CompatibilityPHP70'
project_themes[] = 'CompatibilityPHP71'
project_themes[] = 'CompatibilityPHP72'
project_themes[] = 'Analyze'
project_themes[] = 'Preferences'
project_themes[] = 'Appinfo'
project_themes[] = 'Appcontent'
project_themes[] = '"Dead code"'
project_themes[] = 'Security'
project_themes[] = 'Custom'

 Default reports to generate
project_reports[] = 'Ambassador'


 where is neo4j host
neo4j_host     = '127.0.0.1'
neo4j_port     = '7777'
neo4j_folder   = 'neo4j'
neo4j_login    = 'neo4j'
neo4j_password = 'oui'
oem@latitude:~/t2$ 
```

**Comment by dotancohen** [comment]
In light of the previous comment, there seem to be two bugs:
1. With invalid PHP paths in the config file, Exakat crashes with no output.
2. When running the `doctor` command, Exakat creates a config file with invalid PHP version paths.

**Comment by exakat** [comment]
I could reproduce the 1), and a fix is being worked on.
I see the second problem, as all comments are removed. 

Both still affecting 1.5.2, and the fix targets the 1.5.3 (aka tomorrow).

**Comment by exakat** [comment]
I'm closing this since we reached version 1.6.2.
